services/app-web/src/components/FileUpload/FileItem/FileItem.test.tsx

```diff
@@ -64,22 +64,44 @@ describe('FileItem component', () => {
 
     beforeEach(() => jest.clearAllMocks())
     it('renders without errors', () => {
```

Comment:
The way the code is written right now, with the duplication in FileItem, I would want to double all the tests because we have double the conditionals.  Test the exact same thing `renderMode=table` because right now the code could easily diverge.  🤷  or just refactor